tools/EMLassemblyline/make_eml.xml

```diff
@@ -0,0 +1,70 @@
+<tool id="makeeml" name="Make EML" version="0.1.0+galaxy0">
+    <description> Create EML from EAL templates</description>
+    <requirements>
+        <requirement type="package" version="4.3.1">r-base</requirement>
+        <requirement type="package" version="3.5.5">r-emlassemblyline</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    mkdir output_template &&
+    cd output_template &&
+    #for $input in $templates
+        ln -s '$input' '${input.element_identifier}.txt' &&
+    #end for
+    find . -type f -empty -print -delete &&
+    Rscript '$__tool_directory__/make_eml.R'
+    ]]></command>
+    <inputs>
+        <param name="templates" type="data_collection"  collection_type="list" format="tabular" label="Upload all EAL templates" help= "Input a data collection." />
+    </inputs>
+    <outputs>
+        <data name="metadataout" format="xml" >

```

Comment:
How many output files are ever generated? If only one file, than this collection is overkill. Simply add a `<data>` tag without discover_datasets.

Reply:
Thanks a lot for the review !
I didn't know this extension existed but it seems super usefull thanks !
There is only one output but we don't know its name and by putting it into <data>instead of <collection> tag it does not create a collection does it ?
But there have been some other issues that I'm trying to correct, the tool now create  an empty XML that didn't appear before

Reply:
You can do the following at the end of your command section if you do not know the name.

```
mv output_template/*.pdf my_output.pdf
```

Then you know the output that you want to collect.

Reply:
Thank you Björn for this tip! Here, it is better to keep the original filename... I will try to see the best manner to do that